docs/install/custom-install/custom-install.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Knative installation files and dependencies"
+linkTitle: "Install files and dependencies"
+weight: 20
+type: "docs"
```

Comment:
If this used to be the files that was deleted please make sure you add in the aliases so that the redirects work

Reply:
Also, I'd use "installation" consistently, rather than "install files"

Reply:
I think we could probably actually just have a title here and no linkTitle and just call it "Installation files"; I don't think it actually includes any dependencies and not sure what is meant by this? Also I don't think there's a need to call out Knative 🙂

Reply:
The files were never published so do not need redirects. I was following the instructions in [#1869](https://github.com/knative/docs/issues/1896) when I gave the page a title but I agree that Installation files would be a more concise title.